doc/apiref.rst

```diff
@@ -58,6 +58,21 @@ the library:
       /* Code specific to version 1.3 and above */
       #endif
 
+Additionally, as of ``2.13``, there are functions to determine the version of
+Jansson at runtime:
```

Comment:
There's `.. versionadded:` directive that you should use for documenting the version in which a feature was added. See e.g. the docs of `json_boolean_value()`

Reply:
Otherwise the PR looks good!